index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styel.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <div class="wrapper">
+        <nav>
+          <a href="#" class="logo">Logo</a>
+          <img class="image" src="Enabled.svg" alt="" />
+        </nav>
+      </div>
+    </header>
+    <div class="header-container">
+      <div class="wrapper">
+        <section>
+          <div class="typography">
+            <h1>
+              I’m Mahdia <br />
+              Glad to see you!
+            </h1>
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <a href="">LET’S CONNECT</a>
```

Comment:
- [ ] As requested by the previous reviewer. Please follow the use of the required font as in the template provided by Figma. Check for setup on [google fonts](https://fonts.google.com/specimen/Poppins)